application/pages/InstanceSettingsPage.ui

```diff
@@ -11,16 +11,7 @@
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
-   <property name="leftMargin">
-    <number>0</number>
-   </property>
-   <property name="topMargin">
-    <number>0</number>
-   </property>
-   <property name="rightMargin">
-    <number>0</number>
-   </property>
-   <property name="bottomMargin">
+   <property name="margin">
```

Comment:
This change seems to be unnecessary. I'm also not sure if Qt 5.4 tools can handle it (I haven't tried).